src/app/components/TransactionsTable/TransactionModal.tsx

```diff
@@ -56,17 +57,43 @@ export default function TransactionModal({
         <div>
           <div className="flex items-center justify-center">
             {getTransactionType(transaction) == "outgoing" ? (
-              <div className="flex justify-center items-center bg-orange-100 dark:bg-orange-950 rounded-full p-3">
-                <PopiconsArrowUpSolid className="w-10 h-10 text-orange-400 dark:text-amber-600 stroke-[1px] stroke-orange-400 dark:stroke-amber-600" />
-              </div>
+              transaction.state === "pending" ? (
+                <div className="flex justify-center items-center bg-orange-100 dark:bg-orange-950 rounded-full p-3 animate-pulse">
```

Comment:
@pavanjoshi914 can you please check the new colors in figma? pending should be blue